pdr_backend/pdr_dashboard/dash_components/plots.py

```diff
@@ -338,3 +361,74 @@ def get_figures_and_metrics(
     figs_metrics.make_figures()
 
     return figs_metrics
+
+
+@enforce_types
+def get_feed_figures(payouts: Optional[List], subscriptions: List) -> FeedModalFigures:
+    """
+    Return figures for a selected feed from the feeds table.
+    """
+
+    # Initialize empty figures with default settings
+    figures = FeedModalFigures()
```

Comment:
why not use FeedModalResult to make its own figures? Like the other results manage their own figure creation based on data? That way, we would have all the objects initialised before in a much simpler way.

Reply:
I looked at the entire file and it should be updated to conform to the other structures